airbyte-cdk/java/airbyte-cdk/db-sources/src/main/kotlin/io/airbyte/cdk/integrations/debezium/internals/DebeziumRecordIterator.kt

```diff
@@ -171,6 +171,13 @@ class DebeziumRecordIterator<T>(
             }
 
             val changeEventWithMetadata = ChangeEventWithMetadata(next)
+
+            // #41647: discard event type with op code 'm'
+            if (!isEventTypeHandled(changeEventWithMetadata)) {
+                LOGGER.info { "Event type not handled: $next" }
```

Comment:
```suggestion
                LOGGER.info { "WAL event type not handled: $next" }
```

Reply:
Yes. It's likely that other DBs have similar use cases as in postgres. Thanks